_data/meltano/utilities/dbt-dry-run/dbt-dry-run.yml

```
@@ -0,0 +1,54 @@
+commands:
+  run:
+    args:
+      args:
+        --target ${DBT_DRY_RUN_TARGET}
+        --verbose
+      definition: |
+        Allows you to run the plugin. Requires that you specify a profile target.
+        Note - Requires you run `dbt-bigquery compile` before executing.
+      executable: dbt_dry_run_invoker
+
+definition: |
+  dbt-dry-run is a meltano plugin based on the awesome tool created by the team at AutoTraderUk.
+  It allows dbt-bigquery users to validate if their models will run (without encuring costs).
+  It leverages bigquery's [dry-run](https://cloud.google.com/bigquery/docs/dry-run-queries) feature.
+
+  **A note**
+  This plugin is at it's most powerful (in our opinion) in CI (we use github actions). It allows us to ensure that our changes will not break production.
+domain_url: https://github.com/autotraderuk/dbt-dry-run
+executable: dbt-dry-run
+keywords:
+- dbt
+- bigquery
+- testing
+- continuous-integrations
+- meltano_edk
+label: dbt-dry-run
+logo_url: /assets/logos/utilities/dbt.png
+maintenance_status: active
+name: dbt-dry-dryn
+namespace: dbt-dry-run
```

```suggestion
label: dbt-bigquery-dry-run
logo_url: /assets/logos/utilities/dbt.png
maintenance_status: active
name: dbt-bigquery-dry-dryn
namespace: dbt_bigquery_dry_run
```

As discussed in office hours, proposing we add `bigquery-` into the name where it will be presented in the Hub. This doesn't require changing the underlying repo but we should rename the file and path to match something like `.../utilities/dbt-bigquery-dry-run/{variant_name}.yml` where variant name might be `autotraderuk` or something else...

---

@aaronsteers I wonder what you think about updating the label to something like `dbt-dry-run (BigQuery Only)` and leaving the name and everything else the same as the repo/package name. It always feels a little funny to me to rename packages, although I know we do it for pipelinewise and airbyte so there is precedence for it.

---

@pnadolny13 re:

> @aaronsteers I wonder what you think about updating the label to something like `dbt-dry-run (BigQuery Only)` and leaving the name and everything else the same as the repo/package name.

That could work. Do I remember correctly that `label` is used in search results and as the page title?

Presumably then, if a `dbt-dry-run` were created for Snowflake, a key difference would be in whether names of future variants collide when adding side-by-side to Meltano. But this is just speculation since we don't know if/when non-BigQuery implementations will come about... 🤔 

But anyway, yeah, I think I'm fine with adding ` (BigQuery)` or ` (BigQuery Only)` to the label if that accomplishes what we need to regarding discoverability to new users.